sectionproperties/pre/sections.py

```diff
@@ -2002,129 +2544,115 @@ class PolygonSection(Geometry):
 
         Mesh generated from the above geometry.
     """
+    outer_points = []
+    inner_points = []
 
-    def __init__(self, d, t, n_sides, r_in=0, n_r=1, rot=0, shift=[0, 0]):
-        """Inits the PolygonSection class."""
+    if n_sides < 3:
+        msg = "n_sides required to be greater than 3 for PolygonSection class"
```

Comment:
The exception message here still refers to PolygonSection class, which is now turned into polygon_section geometry function

Change to something like this?
```suggestion
        msg = "n_sides required to be greater than 3 for polygon section type"
```

Reply:
Fixed!